app/scripts/lib/app-start.js

```diff
@@ -130,9 +132,7 @@ function (
         this.initializeInterTabChannel()
       ])
       .then(_.bind(this.allResourcesReady, this))
-      .then(function () {
-        self._trackWindowOnError();
-      }, function (err) {
+      .then(null, function (err) {
```

Comment:
@zaach p-promise does not have `fail`. https://github.com/rkatic/p#api 

I think I tried it a while ago...